### PR TITLE
fix: Fix did not auto delete old log issue.

### PR DESCRIPTION
**Pull request by re2zero**
Did not parse the correct datetime from the log file name.

Log: fix log saver issue.

### CODE REVIEW

**Review by wangrong1069 — APPROVED** [review]
looks good!

**Review by wangrong1069 — APPROVED** [review]
looks good!

File: src/server/backend/lib/logsaver.cpp [hunk 1]
```diff
@@ -125,26 +127,27 @@ void LogSaverPrivate::backupLog()
...
+    for (QFileInfo f: fileList) {
         // "."和".."跳过
         if (f.baseName() == "")
             continue;
 
-        dateTime2 = QDateTime::fromString(f.baseName(), "yyyy-MM-dd");
-        if (dateTime2 < dateTime1) { // 只要日志时间小于前x天的时间就删除
-            dir.remove(f.absoluteFilePath());
+        // 从日志文件后缀获取创建时间（例如 app.log.2023-04-14-00-04-12 > 2023-04-14-00-04-12）
+        dateTime2 = QDateTime::fromString(f.suffix(), "yyyy-MM-dd-hh-mm-ss");
```
Comment (max-lvs) [inline src/server/backend/lib/logsaver.cpp:148]: 从命名规则来看 QDateTime dateTime2 = QDateTime::fromString(f.suffix(), "yyyy-MM-dd-hh-mm-ss");
这样写比较好，139 行是多余的

**Review by wangrong1069 — APPROVED** [review]
looks good!

**Review by max-lvs — APPROVED** [review]

**Pull request merged**